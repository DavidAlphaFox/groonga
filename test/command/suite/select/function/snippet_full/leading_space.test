table_create Entries TABLE_NO_KEY
column_create Entries content COLUMN_SCALAR ShortText

load --table Entries
[
{"content": "groonga and MySQL"}
]

select Entries \
  --output_columns ' \
  snippet_full(content, {"skip_leading_spaces": false}, \
  "MySQL", "<span class=\\"keyword\\">", "</span>" \
  )' \
  --command_version 2
