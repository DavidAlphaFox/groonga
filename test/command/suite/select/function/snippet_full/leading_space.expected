table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Entries
[
{"content": "groonga and MySQL"}
]
[[0,0.0,0.0],1]
select Entries   --output_columns '   snippet_full(content, {"skip_leading_spaces": false},   "MySQL", "<span class=\\"keyword\\">", "</span>"   )'   --command_version 2
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "snippet_full",
          "null"
        ]
      ],
      [
        [
          "groonga and<span class=\"keyword\"> MySQL</span>"
        ]
      ]
    ]
  ]
]
